test/FTL.sql

```diff
@@ -0,0 +1,152 @@
+CREATE TABLE ftl
+(
+    id    INTEGER PRIMARY KEY NOT NULL,
+    value BLOB                NOT NULL
+);
+
+CREATE TABLE counters
+(
+    id    INTEGER PRIMARY KEY NOT NULL,
+    value INTEGER             NOT NULL
+);
+
+CREATE TABLE network
+(
+    id         INTEGER PRIMARY KEY NOT NULL,
+    ip         TEXT                NOT NULL,
+    hwaddr     TEXT                NOT NULL,
+    interface  TEXT                NOT NULL,
+    name       TEXT,
+    firstSeen  INTEGER             NOT NULL,
+    lastQuery  INTEGER             NOT NULL,
+    numQueries INTEGER             NOT NULL,
+    macVendor  TEXT
+);
+
+CREATE TABLE queries
+(
+    id        INTEGER PRIMARY KEY AUTOINCREMENT,
+    timestamp INTEGER NOT NULL,
+    type      INTEGER NOT NULL,
+    status    INTEGER NOT NULL,
+    domain    TEXT    NOT NULL,
+    client    TEXT    NOT NULL,
+    forward   TEXT
+);
+
+CREATE INDEX idx_queries_timestamps ON queries (timestamp);
+
+INSERT INTO ftl
+VALUES
+    -- Version
+    (0, 3),
+    -- Last query timestamp
+    (1, 177180),
+    -- First counter timestamp
+    (2, 0);
```

Comment:
Both, 1 and 2, should be valid timestamps.
```suggestion
    (2, 1556047936);
```

Reply:
These are valid timestamps. They match the query data.

Reply:
First counter timestamp:
```
0 -> GMT: Thursday, 1. January 1970 00:00:00
```

Last query timestamp:
```
177180 -> GMT: Saturday, 3. January 1970 01:13:00
```

Reply:
That being said, it doesn't matter all that much what is in these fields. These have rather historical meanings when we were still importing history from `pihole-FTL.log` to avoid double importing and interpreting the same queries.

I left them there as I felt they might be useful for when we want to show since when a database file is used (`select min(timestamp)` cannot give this information as old queries are typcially deleted after one year).

Reply:
What is the issue with those timestamps? They match the minimum and maximum timestamps from the query data. Yes, they represent very old dates, but that shouldn't matter.

Reply:
Okay, I was assuming that we'd use fairly recent test data, but when this is intended, then that's okay.